Kernel/Graphics/Intel/Transcoder/DisplayTranscoder.cpp

```diff
@@ -5,17 +5,11 @@
  */
 
 #include <Kernel/Arch/x86/IO.h>
-#include <Kernel/Graphics/Intel/DisplayTranscoder.h>
+#include <Kernel/Graphics/Intel/Transcoder/DisplayTranscoder.h>
```

Comment:
Can we put DisplayTranscoder in the Transcoder/ directory when it's first added? That way we can avoid some feng shui in ` Kernel/Graphics: Prepare to support future Intel generation transcoders `